### ISSUE TITLE
Package.swift is not available in the repository

### ISSUE THREAD

**Issue by MaxDesiatov** [flowkey/SDL2-SwiftPackageManager#6]
Maybe I'm missing something, but what exactly would make this repository compatible with SwiftPM? There's no `Package.swift` file anywhere, so how one would add it as a dependency in their own package manifest?

**Comment by ephemer** [comment]
Hi @MaxDesiatov you'd need to create a Package.swift based on the CMake files we're using. If you clone the repo and look in the git history you should be able to find the one we used to use.

We moved away from SwiftPM because we couldn't get it working cross-platform with Swift 5 and have had years of battling with it otherwise. We might consider moving back to SwiftPM if the cross-compiling story improves for our use case

**Comment by MaxDesiatov** [comment]
Thanks for the clarification. Would the repository name and `README.md` be updated until then? Both of those seem to be misleading at the moment.

**Comment by ephemer** [comment]
True. To be clear we _are_ using this in Swift, for https://github.com/flowkey/uikit-cross-platform, but didn't think anyone else was aware of this repo specifically.

**Comment by ephemer** [comment]
@MaxDesiatov if you want to use it with SwiftPM, the suggestion to look in the git history still stands. We removed the old one in ab962c68b37cae017de42043bf983e3564480718 - it shouldn't take much to get that working again
